vue-frontend/src/components/mobile-app-development/CTASection.vue

```diff
@@ -0,0 +1,113 @@
+      <img
+        :src="background400w"
+        :srcset="`${background400w} 400w, ${background800w} 800w`"
+        loading="lazy"
+        class="tw-absolute tw-h-full tw-w-full"
+        alt="background-image"
+      />
+
+      <div
+        class="tw-container tw-flex tw-flex-col tw-items-center tw-justify-center tw-py-[5rem]"
+      >
+        <p
+          class="v2-canopas-gradient-text tw-text-center tw-font-inter-medium tw-text-[1rem] tw-leading-[1.21rem]"
+        >
+          Get started today
+        </p>
+        <p
+          class="tw-mt-6 tw-text-center tw-font-inter-bold tw-text-[2.25rem] tw-leading-[2.723rem] tw-text-black-core/[0.85]"
+        >
+          Let's build the next big thing!
+        </p>
+        <p
+          class="tw-mt-4 tw-text-center tw-font-inter-regular tw-text-[0.875rem] tw-leading-[1.313rem] tw-text-black-core/[0.75]"
+        >
+          Let's improve your business's digital strategy and implement robust
+          mobile apps to achieve your business objectives. Schedule Your Free
+          Consultation Now.
+        </p>
+        <router-link
+          class="gradient-btn consultation-btn tw-mt-10 tw-flex tw-w-max tw-items-center tw-rounded-full tw-py-[1.25rem] tw-text-center"
+          :to="contactURL"
+        >
+          <span
+            class="tw-mr-2.5 tw-font-inter-semibold tw-text-[1.1875rem] tw-leading-[1.1875rem] md:tw-text-[1.09375rem] md:tw-leading-[1.3125rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.4375rem]"
+            >Get Free Consultation
+          </span>
+        </router-link>
+      </div>
+    </div>
+    <!-- MOBILE UI END -->
+
+    <!-- DESKTOP UI START -->
+    <div class="tw-hidden md:tw-block">
+      <img
+        :src="desktop_background800w"
+        :srcset="`${desktop_background800w} 800w, ${desktop_background1200w} 1200w, ${desktop_background1600w} 1600w, ${desktop_background2400w} 2400w `"
+        loading="lazy"
+        class="tw-absolute tw-h-full tw-w-full"
+        alt="background-image"
+      />
+      <div
+        class="tw-container tw-z-[1] tw-flex tw-flex-col tw-items-center tw-justify-center tw-py-[5rem]"
+      >
+        <p
+          class="v2-canopas-gradient-text tw-text-center tw-font-inter-medium tw-text-[1.125rem] tw-leading-[1.375rem] xl:tw-text-[1.25rem] xl:tw-leading-[1.513rem]"
+        >
+          Get started today
+        </p>
+        <p
+          class="tw-mt-4 tw-text-center tw-font-inter-bold tw-text-[2.813rem] tw-leading-[3.438rem] tw-text-black-core/[0.85] xl:tw-text-[3.438rem] xl:tw-leading-[4.16rem]"
+        >
+          Let's build the next big thing!
+        </p>
+        <p
+          class="tw-mt-4 tw-w-[75%] tw-text-center tw-font-inter-regular tw-text-[1.063rem] tw-leading-[1.625rem] tw-text-black-core/[0.75] lg:tw-w-full xl:tw-text-[1.25rem] xl:tw-leading-[1.875rem]"
+        >
+          Let's improve your business's digital strategy and implement robust
+          mobile apps<br />
```

Comment:
It is not looking effective on some widths

![Screenshot from 2023-06-27 13-15-32](https://github.com/canopas/canopas-website/assets/69897605/97d71c39-1db2-45a9-97c7-e1c1d90d20dd)